docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/migrate-rancher-to-new-cluster.md

```diff
@@ -20,10 +21,10 @@ It is required to use the same hostname that was set as the server URL in the fi
 
 Rancher version must be v2.5.0 and up
 
-Rancher can be installed on any Kubernetes cluster, including hosted Kubernetes clusters such as Amazon EKS clusters. For help installing Kubernetes, refer to the documentation of the Kubernetes distribution. One of Rancher's Kubernetes distributions may also be used:
+Rancher can be installed on any Kubernetes cluster, including hosted Kubernetes clusters such as Amazon EKS clusters. For help installing Kubernetes, refer to the documentation of the Kubernetes distribution.
+
+Since Rancher can be installed on any Kubernetes cluster, you can use this backup and restore method to migrate Rancher from one Kubernetes cluster to any other Kubernetes cluster. This method *only* migrates Rancher-related resources and won't affect other applications on the cluster. Refer to the [support matrix](https://www.suse.com/lifecycle/) to identify which Kubernetes cluster types and versions are supported for your Rancher version. 
```

Comment:
If the dates on the /lifecycle page aren't relevant, we should link directly to the support matrix page to reduce the pages the user needs to navigate to. In that case, we can also link to the associated matrix for each version of the docs.

Reply:
The support matrix is per Rancher Manager version, so we can't link directly to any specific version..

Reply:
For my first point I meant linking to https://www.suse.com/suse-rancher/support-matrix/all-supported-versions rather than adding another hop through https://www.suse.com/lifecycle/ and having the user try and find the matrix link. 

For my second point, the matrix link on https://www.suse.com/lifecycle/ goes to the 2.7.0 matrix (due to the constraint you mentioned). Within our docs we have different versions of this page so:

- On the v2.5 version of this page the current link takes users to the 2.7.0 matrix. My suggestion is to use https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-5-0/ (or the most recent 2.5) for the v2.5 version of this page so the link takes users to the general 2.5 area of the matrix page.
- Similarly the v2.6 version of this page the current link takes users to the 2.7.0 matrix. Similarly using https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-0/ (or the most recent 2.6) for the v2.6 version of this page so the link takes users to the general 2.6 area of the matrix page.